lib/utils/request.ts

```diff
@@ -32,10 +51,13 @@ export const requestWithNodeHttpModule: RequestFnType = (url, data) => {
 
   const { request: nodeRequest } =
     url.indexOf("https://") === 0 ? require("https") : require("http");
-  const req = nodeRequest(options);
+  const req = nodeRequest(reqOpts);
 
   req.on("error", error => {
     console.error(error);
+    if (options && options.errorCallback) {
+      options.errorCallback(error);
+    }
   });
```

Comment:
@sarahdayan  This part bothers me: we already have a `console.error` in here, so it's not consistent with the XHR requester.
That's what pushed me to suggest benefiting from the breaking change coming with next PR to just add the same `console.error` in the XHR error handler.
Wdyt?

(you can answer in the PR conversation instead of here if you wish, so the discussion isn't duplicated)